manifests/0000_50_cluster-monitoring-operator_02-role.yaml

```diff
@@ -113,11 +113,11 @@ rules:
   - securitycontextconstraints
   verbs:
   - create
-  - delete
   - get
   - list
-  - update
   - watch
+  - update
+  - delete
```

Comment:
/lgtm
not sure why it changed the ordering?

Reply:
:man_shrugging: